From 75bee41cc4700917a7c39bc9dcddf8e96e0d4386 Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Thu, 30 Aug 2018 18:46:17 -0400
Subject: [PATCH 236/249] nash: Link a custom TWRP image

* Until TWRP build servers figure out why update_engine*
  isn't being built on android-8.1 branches.

Change-Id: Ic2f656f3f1152d41037c7bd7d05c8414a055692c
---
 _data/devices/nash.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/_data/devices/nash.yml b/_data/devices/nash.yml
index 005bf52..1c8eecb 100644
--- a/_data/devices/nash.yml
+++ b/_data/devices/nash.yml
@@ -12,6 +12,7 @@ cpu: Kryo
 cpu_cores: '8'
 cpu_freq: 4 x 2.45 GHz + 4 x 1.9 GHz
 current_branch: 15.1
+custom_twrp_link: https://androidfilehost.com/?fid=3700668719832238233
 depth: 6.1 mm (0.24 in)
 download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 540
-- 
2.17.1

